aries-site/src/examples/templates/wizard/WizardAnatomy.js

```diff
@@ -0,0 +1,219 @@
+import React, { useContext, useEffect, useMemo, useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import { ThemeContext } from 'styled-components';
+import { Box, Diagram, Grid, Form, ResponsiveContext, Stack } from 'grommet';
+import {
+  CancellationLayer,
+  StepFooter,
+  StepHeader,
+  WizardContext,
+  WizardHeader,
+} from './components';
+import { getWidth } from './utils';
+import { Annotation } from '../../../layouts';
+import { StepOne } from './WizardValidationExample';
+import { connection } from '../../../utils/utils';
+
+const connections = [
+  connection('1a', 'wizard-title', 'vertical'),
+  connection('1b', 'previous-button'),
+  connection('1c', 'cancel-button'),
+  connection('2b', 'step-summary'),
+  connection('2a', 'step-title'),
+  connection('2c', 'step-description'),
+  connection('3', 'step-inputs'),
+  connection('4', 'step-footer'),
+  connection('4a', 'next-step'),
+];
+
+export const WizardAnatomy = ({ containerRef }) => (
+  <Stack interactiveChild="first">
+    <Grid
+      columns={['xxsmall', 'flex', 'xxsmall']}
+      rows={['36px', '60px', '48px', '24px', '60px', '60px', 'auto', '85px']}
+      areas={[
+        ['empty-1', 'annotation-1a', 'empty-2'],
+        ['annotation-1b', 'wizard', 'annotation-1c'],
+        ['empty-3', 'wizard', 'empty-4'],
+        ['annotation-2b', 'wizard', 'empty-4'],
+        ['annotation-2a', 'wizard', 'empty-4'],
+        ['annotation-2c', 'wizard', 'empty-4'],
+        ['annotation-3', 'wizard', 'empty-4'],
+        ['annotation-4', 'wizard', 'annotation-4a'],
+      ]}
+      align="center"
+      justify="center"
+      gap={{ column: 'xsmall' }}
```

Comment:
If we apply `fill` to the WizardExample, we probably want to change increase this gap.

```suggestion
      gap={{ column: 'medium' }}
```

Reply:
Incorporated in latest commit.